.travis.yml

```diff
@@ -1,3 +1,4 @@
+version: ~> 1.0
```

Comment:
I looked this up -- https://docs.travis-ci.com/user/build-config-validation -- and it looks cool, but I think this belongs in a separate PR, unless it has something to do with Python versions that I don't see.

Reply:
I needed it to figure out what was happening under the covers with Travis. Otherwise, I was having some silent failures. So it's connected in the sense that it gets the rest of the code here.

Reply:
removed for now